packages/flutter_tools/gradle/src/test/kotlin/tasks/DeepLinkJsonFromManifestTaskTest.kt

```diff
@@ -90,68 +95,50 @@ class DeepLinkJsonFromManifestTaskTest {
 
     @Test
     fun applicationNoDeepLinkingElements() {
-        val manifestContent = """
-            <?xml version="1.0" encoding="utf-8"?>
-            <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-                package="$defaultNamespace">
-                <application android:label="Test App" android:icon="@mipmap/ic_launcher">
-                </application>
-            </manifest>
-            """
+        val manifestContent = DeeplinkManifestBuilder().build()
         val manifestFile = createTempManifestFile(manifestContent)
-        val appLinkSettings = DeepLinkJsonFromManifestTaskHelper.createAppLinkSettings(defaultNamespace, manifestFile)
-
-        assertEquals(defaultNamespace, appLinkSettings.applicationId)
-        assertFalse(appLinkSettings.deeplinkingFlagEnabled)
-        assertTrue(appLinkSettings.deeplinks.isEmpty())
+        try {
+            val appLinkSettings = DeepLinkJsonFromManifestTaskHelper.createAppLinkSettings(defaultNamespace, manifestFile)
+            assertEquals(defaultNamespace, appLinkSettings.applicationId)
+            assertFalse(appLinkSettings.deeplinkingFlagEnabled)
+            assertTrue(appLinkSettings.deeplinks.isEmpty())
```

Comment:
Nit: Is the `SAXParseException` so unclear that it deserves a try/catch? Just curious because I'm a bit worried that someone may try to copy the try/catch pattern you use here, forget to not just specifically catch `SAXParseException`s, and then accidentally swallow assertion errors from the assertion tests.

Reply:
Yeah it is super unhelpful and gives you basically nothing to debug with. I kept it focused on the specific exception that happens when the xml is invalid because the xml is being created by test tooling.